recipes/pysdl2/locate_sdl.patch

```diff
@@ -0,0 +1,18 @@
+--- sdl2/__init__.py
++++ sdl2/__init__.py
+@@ -1,4 +1,15 @@
+ """SDL2 wrapper package"""
++# Direct this module to Ana/Miniconda's lib folder, where the SDL2 libraries
++# can be found.
++import os
++
++if os.name == "posix":
++	os.environ["PYSDL2_DLL_PATH"] = os.path.abspath(os.path.join(
++		os.path.dirname(__file__), os.pardir, os.pardir, os.pardir))
++elif os.name == "nt":
++	os.environ["PYSDL2_DLL_PATH"] = os.path.abspath(os.path.join(
++		os.path.dirname(__file__), os.pardir, os.pardir, os.pardir, "Library", "bin"))
```

Comment:
I think option 1 is not possible. If not supplied with a PYSDL2_DLL_PATH,  pysdl2 uses the `ctypes.util.find_library()` function to locate the sdl2 library, which as far as I know cannot be told to use a prefix for search. I think the option to hardcode PYSDL2_DLL_PATH is exactly meant by pysdl2's devlopers to solve this shortcoming. I sadly don't really have the time to dive this deep into integrating a more elegant solution in pysdl2's source myself to make it use a prefix for search. 

Option 2 is possible, but I predict this will often result in undesired behavior.
`ctypes.util.find_library()` has different search behaviors on the various OS's. According to the [docs](https://docs.python.org/2/library/ctypes.html#finding-shared-libraries) of this function:
- On Linux, find_library() tries to run external programs (/sbin/ldconfig, gcc, and objdump) to find the library file. It returns the filename of the library file. If conda doesn't catch this call to ldconfig, a system library will be returned.
- On OS X, find_library() tries several predefined naming schemes and paths to locate the library, and returns a full pathname if successful.
- On Windows, find_library() searches along the system search path, and returns the full pathname, but since there is no predefined naming scheme a call like find_library("c") will fail and return None.
So unless conda itself has modified ctypes in such a way that it only looks inside conda when find_library is called, there is no way to tell which sdl2 library will be picked up and used, and thus no way to guarantee the conda supplied sdl2 libraries are used.